packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -148,7 +158,42 @@ const useBigNumberConfig = (
               compact: bigNumberStyle,
           });
 
-    const showStyle = isNumber && (!isField(item) || item.format !== 'percent');
+    const formatValues = {
+        RAW: 'raw',
+        PERCENTAGE: 'percentage',
+    };
+    const [showComparison, setShowComparison] = useState<boolean>(false);
+    const [comparisonFormat, setComparisonFormat] = useState<string>(
+        formatValues.RAW,
+    );
+
+    const calculateComparisonValue = (a: number, b: number, format: string) => {
+        const rawValue = Math.round(a - b);
+        switch (format) {
+            case formatValues.PERCENTAGE:
+                return Math.round(((a - b) / b) * 100);
+
+            case formatValues.RAW:
+                return rawValue === 0 ? 'No change' : rawValue;
+            default:
+                return rawValue;
+        }
+    };
+
+    const comparisonValue =
+        firstRowValueRaw &&
+        secondRowValueRaw &&
```

Comment:
Be careful when checking number values like this since `0` is considered false. Checking for `isFirstRowValueNumber` and `isSecondRowValueNumber` should be enough to confirm we can compare the values.